pygmt/src/solar.py

```diff
@@ -50,7 +50,9 @@ def solar(self, terminator="d", terminator_datetime=None, **kwargs):
         passed as a string or Python datetime object.
         [Default is the current UTC date and time]
     {R}
+        *Required if this is the first plot command.*
     {J}
+        *Required if this is the first plot command.*
```

Comment:
`-J` and `-R` are not required for `solar`, because the following CLI command works:
```
gmt solar -W1p -Tdc -pdf map
```

Reply:
fixed in fd18364